tbx/project_styleguide/templates/patterns/molecules/help-block/help-block.html

```diff
@@ -1,51 +1,34 @@
 {% load wagtailcore_tags wagtailimages_tags %}
-<div class="help-block">
-    <div class="help-block__container">
-        <section class="help-block__content">
-            <div class="help-block__section-title">
-                {% include "patterns/atoms/section-title/section-title.html" with title=title %}
-            </div>
-            <h2 class="help-block__title">
-                {{ intro|richtext }}
-            </h2>
-            <div class="help-block__links-container">
-                <ul class="help-block__link-list">
-                    {% for item in items %}
-                        {% include "patterns/molecules/list-item/list-item.html" with item=item %}
-                    {% endfor %}
-                </ul>
-            </div>
-        </section>
+{% include "patterns/atoms/section-title/section-title.html" with title=title %}
+<h2>
+    {{ intro|richtext }}
+</h2>
```

Comment:
I'm curious if this works? Would have thought as it has the `|richtext` filter it would leave the `h2` tag empty. (appreciate this was already like this from the previous build)

Reply:
It puts a p tag inside the h2 so it does work, but is admittedly odd - I'll just remove the h2.

Reply:
Ah good to know!